LDKNodeMonday/View Model/Home/OnboardingViewModel.swift

```diff
@@ -104,10 +108,10 @@ class OnboardingViewModel: ObservableObject {
             let backupInfo = BackupInfo(mnemonic: seedPhrase)
             try KeyClient.live.saveBackupInfo(backupInfo)
             try KeyClient.live.saveNetwork(selectedNetwork.description)
-            try KeyClient.live.saveEsploraURL(selectedURL)
+            try KeyClient.live.saveEsploraURL(selectedEsploraServer.url)
             LightningNodeService.shared = LightningNodeService()
             DispatchQueue.main.async {
-                self.isOnboarding = false
+                UserDefaults.standard.set(false, forKey: "isOnboarding")
```

Comment:
note: this will change back to `self.isOnboarding = false` when you change back to ObservableObject